packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -672,7 +672,7 @@ const tenantsRouter = (
       const ctx = fromAppContext(req.ctx);
 
       try {
-        validateAuthorization(ctx, [ADMIN_ROLE, M2M_ROLE]);
+        validateAuthorization(ctx, [ADMIN_ROLE, M2M_ROLE, M2M_ADMIN_ROLE]);
```

Comment:
The tenant process `addCertfieidAttribute` route has `M2M_ROLE` authorization. 
Does it really need it?